racetrack_commons/racetrack_commons/auth/auth.py

```diff
@@ -4,7 +4,7 @@
 class AuthSubjectType(Enum):
     USER = 'user'
     ESC = 'esc'
-    FATMAN_FAMILY = 'fatman_family'
+    JOB_FAMILY = 'job_family'
```

Comment:
Let's hold on with that. This enum value is baked in the existing JWT auth tokens and it might be troublesome if we change it now. I'll make it in a separate issue.
```suggestion
    JOB_FAMILY = 'fatman_family'
```